WorkloadManagementSystem/Service/JobStateUpdateHandler.py

```diff
@@ -183,27 +183,38 @@ def export_setJobStatusBulk(self, jobID, statusDict):
     lastTime = Time.toString(Time.fromEpoch(lastTime))
 
     dates = sorted(statusDict)
+    log = self.log.getSubLogger('JobStatusBulk/Job-%s' % jobID)
+    log.verbose("*** New call *** Last update time %s - Sorted new times %s" % (lastTime, dates))
```

Comment:
Please make this a `debug`, or change it to `log.verbose("Setting bulk status", "Last update time %s - Sorted new times %s" % (lastTime, dates))`